pages/en/lb3/Embedded-models-and-relations.md

```diff
@@ -135,12 +137,10 @@ For example, a Customer can have multiple email addresses and each email address
 ```
 
 {% include important.html content="
-
 Treat `embedsMany` as an actual relation, no different from `hasMany`, for example.
 This means that you cannot just POST the full object with embedded/nested data to create everything all at once.
 So using the example above to add a Customer and multiple email addresses would require two POST operations,
 one for the Customer record and one for the multiple email address data.
```

Comment:
There's an issue related to this section:
https://github.com/strongloop-internal/scrum-apex/issues/123

It might be worth coordinating with @virkt25 on this to make sure that this is more accurate.